entities/src/main/java/com/flipkart/varadhi/entities/ProducerMessage.java

```diff
@@ -0,0 +1,56 @@
+package com.flipkart.varadhi.entities;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.Multimap;
+
+import java.util.List;
+
+public class ProducerMessage implements Message {
```

Comment:
should this be moved to messaging.produce (PulsarProducer test might require a test fixture in entities for this)?

Reply:
PulsarProducerTest is able to access producer message via the entities dependency

Reply:
ok, suggestion was to check if this can be moved to messaging.produce as this is specific to produce path alone. ?